docs/install.rst

```diff
@@ -107,3 +107,6 @@ Once installed, include the ``--network hardhat`` flag to run Brownie with Hardh
         brownie console --network hardhat
 
 The first time you use Hardhat within a Brownie project, a ``hardhat.config.js`` `configuration file <https://hardhat.org/config/>`_ is generated. You should not modify any of the settings within this file as they are required for compatibility.
+
+
+If you have updated your brownie version from an older version you will not have the latest `~/.brownie/network-config.yaml` file, it can be updated using the one [here](https://github.com/eth-brownie/brownie/blob/master/brownie/data/network-config.yaml)
```

Comment:
This is using markdown syntax and will not render correctly. Can you redo it using RST?

Reply:
Alright!

Reply:
done!